GitExtUtils/GitArgumentBuilder.cs

```diff
@@ -174,12 +175,20 @@ public void Add(GitConfigItem configItem)
         public override string ToString()
         {
             var arguments = base.ToString();
+            var gitArgsLength = _gitArgs?.Length;
```

Comment:
why not `_gitArgs?.Length ?? -1`, this saves from doing the null check twice below